index.ts

```diff
@@ -88,15 +88,20 @@ addTests('isEnterprise', [
 	'https://not-github.com/',
 	'https://my-little-hub.com/',
 	'https://my-little-hub.com/gist',
+	'https://my-little-hub.com/gist/in-fragrante',
 ]);
 
-export const isGist = (url: URL | HTMLAnchorElement | Location = location): boolean => url.hostname.startsWith('gist.') || url.pathname.split('/', 2)[1] === 'gist';
+export const isGist = (url: URL | HTMLAnchorElement | Location = location): boolean => typeof getCleanGistPathname(url) === 'string';
```

Comment:
This test was incorrect. It would match `gist.any.com` while in reality GHE probably only supports `/gist/`

Reply:
I remember we had a link to a github enterprise that we were able to test things out.

Reply:
I was wrong:

https://docs.github.com/en/enterprise-server@3.4/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists#about-gists


> `http(s)://[hostname]/gist` or `http(s)://gist.[hostname]` if subdomains are enabled.